### ISSUE TITLE
Hello, the Evaluation metric obtained by my trained network is quite different from the data on the paper. Could you please upload the trained parameters?

### ISSUE THREAD

**Issue by wanliliuxiansen** [stefanLeong/S2CRNet#5]

**Comment by vinthony** [comment]
Hi, I create a new demo repo and it contains the pre-trained weights of our S2CRNet-SqueezeNet, please run the demo here https://github.com/vinthony/S2CRNet-demos

**Comment by wanliliuxiansen** [comment]
Sorry，the demo.ipynb file is bad,please update it.

**Comment by vinthony** [comment]
https://colab.research.google.com/drive/12gH5XDhpCGS_B_5xaJtCVvYrw6gfHtaS#scrollTo=Y5vFGrBLctND you can run it through google colab online
